reviews-receiver/src/producer.ts

```diff
@@ -0,0 +1,49 @@
+import { IRawReview } from './../../shared-types/types';
+import { Producer, Kafka } from 'kafkajs';
+import winston from 'winston';
+import createLogger from './utils/logger';
+
+class ReviewsProducer {
+  private kafkaProducer: Producer;
+  private logger: winston.Logger;
+
+  public constructor() {
+    this.kafkaProducer = this.createKafkaProducer();
+    this.logger = createLogger('producer');
+  }
+
+  private createKafkaProducer(): Producer {
+    const brokers = process.env.KAFKA_BROKERS?.split(',') || ['localhost:9094'];
```

Comment:
take brokers from config

Reply:
I did the same as your reviews-loader consumer to make it pretty similar

Reply:
its ok as a fallback but you can use a `config.ts` file (like [here](https://github.com/RevisarAI/RevisarAI/blob/dev/backend/src/config.ts)) to load the environment variables and also set a fallback value with `.default()`